### PR TITLE
More patterns

### PR DESCRIPTION
**Pull request by DFurnes**
# Changes
 - Adds list-compacted pattern from the DS app.
 - Adds extensions to info-bar (dark variation) and media pattern (medium size) from DS app.

### CODE REVIEW

File: scss/_components/_list-compacted.scss [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+// or for cases where list markup is not directly editable (e.g. Markdown).
+//
+// Styleguide List Compacted
+.list-compacted {
+  li + li {
+    margin-top: 0;
+  }
+}
+
+.list-compacted-wrapper {
```
Comment (weerd) [inline scss/_components/_list-compacted.scss]: <http://nooooooooooooooo.com/>

Comment (DFurnes) [inline scss/_components/_list-compacted.scss]: We talked about this before, you monster.

Comment (weerd) [inline scss/_components/_list-compacted.scss]: :japanese_goblin: doesn't mean I like it, haha

**Comment by weerd** [comment]
:+1:

**Pull request merged**